tempesta_fw/t/unit/test_hash.c

```diff
@@ -34,12 +34,12 @@ TEST(tfw_hash_str, calcs_diff_hash_for_diff_str)
 	 * The hashes should be different with high probability,
 	 * but at this point we are not going to write some statistical tests.
 	 */
-	TfwStr s1 = { .len = 10, .ptr = (void *)"foobarbaz1" };
-	TfwStr s2 = { .len = 10, .ptr = (void *)"Foobarbaz1" };
-	TfwStr s3 = { .len = 10, .ptr = (void *)"foobarbaz2" };
-	TfwStr s4 = { .len = 9, .ptr = (void *)"foobarbaz" };
-	TfwStr s5 = { .len = 11, .ptr = (void *)"foobarbaz11" };
-	TfwStr s6 = { .len = 0, .ptr = (void *)"" };
+	TfwStr s1 = { .len = 10, .data = (void *)"foobarbaz1" };
+	TfwStr s2 = { .len = 10, .data = (void *)"Foobarbaz1" };
```

Comment:
Why don't you use `DEFINE_TFW_STR()` macro for all cases like this one? That's easier, clearer, and avoids potential errors with the lengths of these strings.